core/trino-grammar/src/main/antlr4/io/trino/grammar/sql/SqlBase.g4

```diff
@@ -117,6 +117,13 @@ statement
     | CALL qualifiedName '(' (callArgument (',' callArgument)*)? ')'   #call
     | CREATE (OR REPLACE)? functionSpecification                       #createFunction
     | DROP FUNCTION (IF EXISTS)? functionDeclaration                   #dropFunction
+    | CREATE (OR REPLACE)? BRANCH (IF NOT EXISTS)? identifier
+        (WITH properties)? IN TABLE qualifiedName                      #createBranch
```

Comment:
Should there be `[FROM other-branch]` here? Or will it be modeled as a property.